charts/kong/values.yaml

```diff
@@ -134,11 +134,16 @@ proxy:
 # namespace as Kong is being installed.
 # The `name` property refers to the name of the ConfigMap or Secret
 # itself, while the pluginName refers to the name of the plugin as it appears
+# Sub dicrectories are also supported, the example below show how they can be
+# mounted.
```

Comment:
Consolidating example explanation, some additional explanation, and some minor spelling/grammar stuff:
> Subdirectories require separate ConfigMaps/Secrets. "path" indicates their directory under the main plugin directory: the example below will mount the contents of kong-plugin-rewriter-migrations at "/opt/kong/rewriter/migrations".